.github/workflows/vm-pr.yml

```diff
@@ -163,6 +163,8 @@ jobs:
 
       - run: npm run test:blockchain -- ${{ matrix.args }}
 
+      - run: npm run test:blockchain:transitionForks
```

Comment:
How come this doesn't trigger the CI?

Reply:
this would be run on every matrix hardfork run, is that what you want?

> How come this doesn't trigger the CI?

not sure what you mean? if you want just one added job you could add to the matrix.args array

Reply:
I don't want to add it to the matrix, since these are super small jobs and they will eat up the concurrent jobs which can run. I have added this new npm rule whichruns these tests (in total it are just 78 tests), but I don't want to spawn 6 micro jobs for those. I'd say that adding this line should add all forks but apparently not? Or does it actually trigger this job multiple times each time after it has run a normal test? So it would first run the London tests and then do the transition forks tests..? (Which is also not clean :( )

Reply:
oh the transition forks are already listed at the end of the allForks command, which is run in the VM nightly. i think the reason we limited the ones run on PR to keep the number of jobs down to the amount we can run concurrently at once.

Reply:
Keep in mind this is only triggered if you add `test all hardforks` label. That's only done on PRs which change EVM logic, which is not the case for most PRs. So I think it is fine to add this here?

Reply:
yeah sure, I can go with that. So I think what you want to do here is generalize the test run command above this line

```run: npm run test:blockchain -- ${{ matrix.args }}```

to

```run: npm run ${{ matrix.args }}```

then you can add testing the transition forks as one more matrix argument, so it will just spawn one job for it. the code right now would run it on every matrix run.

Reply:
Oh yeah of course :man_facepalming: Thanks :smile: :+1: